Classes/Emogrifier.php

```diff
@@ -316,6 +348,26 @@ function (array $m) {
     }
 
     /**
+     * Disable the parsing of inline styles
+     *
+     * @see $isInlineStylesParsingEnabled
```

Comment:
I don't think we need this @see as the name of the field is not relevant to the external API.